restconnector/RequestHandler.cpp

```diff
@@ -25,9 +27,13 @@ RequestHandler::RequestHandler( const std::string& publisherSchema,
     , publisher_( new ::zeq::Publisher( lunchbox::URI( publisherSchema ) ) )
     , listening_( true )
     , listeningThread_( new boost::thread( boost::bind( &RequestHandler::listen_, this ) ) )
+    , renderingRessourceStrarted_( false )
 {
-    subscriber_->registerHandler( ::zeq::hbp::EVENT_IMAGEJPEG,
-                                  boost::bind( &RequestHandler::onImageJPEGEvent_, this, _1 ));
+    ZeqEventDescriptor eventDescriptor;
+    eventDescriptor.zeqEvent_ = ::zeq::hbp::EVENT_IMAGEJPEG;
+    eventDescriptor.schema_ = ::zeq::hbp::SCHEMA_IMAGEJPEG;
+    addEventDescriptor_( ::zeq::hbp::IMAGEJPEG, eventDescriptor );
+
     subscriber_->registerHandler( ::zeq::hbp::EVENT_HEARTBEAT,
```

Comment:
What do you mean by generic? EVENT_HEARTBEAT differs from other events, it is why it has it's own handler. onKnowEvent() process events sended to the client ( serialize in JSON ).